extra/backup-tasks.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Are this files use by the code somewhere else in the checks? If not, I would rather remove them for now